lib/GLPI/Agent/Task/Inventory/Generic/Remote_Mgmt/MeshCentral.pm

```diff
@@ -0,0 +1,99 @@
+package GLPI::Agent::Task::Inventory::Generic::Remote_Mgmt::MeshCentral;
+
+use strict;
+use warnings;
+
+use parent 'GLPI::Agent::Task::Inventory::Module';
+
+use English qw(-no_match_vars);
+
+use GLPI::Agent::Tools;
+
+sub isEnabled {
+    my (%params) = @_;
+    my $logger    = $params{logger};
+
+    if (OSNAME eq 'MSWin32') {
+
+        GLPI::Agent::Tools::Win32->use();
+
+        my $key = getRegistryKey(
+            path        => "HKEY_LOCAL_MACHINE/SOFTWARE/Open Source/Mesh Agent",
+            required    => [ qw/NodeId/ ],
+            maxdepth    => 1,
+            logger      => $params{logger}
+        );
+
+        return $key && (keys %$key);
+
+    } elsif (OSNAME eq 'darwin') {
+        return canRun('defaults') && Glob(
+            "/Library/LaunchDaemons/meshagent*.plist"
+        );
+    }
+
+    return Glob('/etc/systemd/system/meshagent.service');
```

Comment:
Sorry, I missed that yesterday.
You should better use has_file() API here and also return unless OSNAME is linux:
```suggestion
    return unless OSNAME eq 'linux';
    return has_file('/etc/systemd/system/meshagent.service');
```